src/main/java/duke/Event.java

```diff
@@ -0,0 +1,58 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Represents an event.
+ */
+public class Event extends Task {
+    private LocalDate date;
+    private Date time = null;
+    SimpleDateFormat timeFormatter = new SimpleDateFormat("h:mm a");
+    private DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("MMM d yyyy");
```

Comment:
Since these two variables are present in more than 1 class, maybe it is possible to consolidate them into one utility class (integrate into Parser perhaps)?